Frameworks/Foundation/NSOperation.mm

```diff
@@ -56,7 +56,9 @@ @implementation NSOperation {
     std::recursive_mutex _completionBlockLock; // guards _completionBlock
 
     std::atomic<NSOperationQueuePriority> _queuePriority;
-    std::atomic<BOOL> _inQueue;
+
+    bool _inQueue;
+    unsigned int _queueIndex;
```

Comment:
nit: I'd prefer `NSUInteger`; it lets us expand seamlessly if we ever widen the `NS*` family of types